### PR TITLE
add button to copy include link for resource

### PR DESCRIPTION
**Pull request by ax-sc**
Add a button for each resource item to copy the respective markdown link.

### CODE REVIEW

**Comment by 3j14** [comment]
We already have a method to copy text (see doku/static/src/components/ui/InlineVariable.vue). Could we just use clipboard.js here instead of using a new package?

**Comment by ax-sc** [comment]
Sorry @jnsdrtlf , I have missed that out. Refactored it to `clipboard.js` now, could you review this again?

File: doku/static/src/components/ui/ResourceItem.vue [hunk 1]
```diff
@@ -25,12 +26,26 @@
...
       AnimatedNotice
     },
+    computed: {
+      includeLink: function() {
+        return '![' + this.resource.name + '](dokures:' + this.resource.filename + ')'
+      }
+    },
+    mounted() {
+      var copyLink = new ClipboardJS('#copy' + String(this.resource.id), {
+        text: () => '![' + this.resource.name + '](dokures:' + this.resource.filename + ')'
```
Comment (3j14) [inline doku/static/src/components/ui/ResourceItem.vue:43]: This could probably be replaced using the computed `this.includeLink`

Comment (3j14) [reply doku/static/src/components/ui/ResourceItem.vue:43]: Minor change which can be done later ;)

**Review by 3j14 — APPROVED** [review]
Looks good to me 👍

**Pull request merged**